### PR TITLE
Sync UI

### PR DESCRIPTION
**Pull request by CodePrometheus**
- [x] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Already closed.
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/docs/en/changes/changes.md).

### CODE REVIEW

**Comment by wu-sheng** [comment]
I just merged one, please update again.

**Review by wu-sheng — COMMENTED** [review]
You should update change logs included rather than only yours.

File: docs/en/changes/changes.md [hunk 1]
```diff
@@ -104,6 +104,9 @@
 * Add the layers filed and associate layers dashboards for the service topology nodes.
 * Fix `Nginx-Instance` metrics to instance level.
 * Update tabs of the Kubernetes service page. 
+* Add Airflow monitor.
```
Comment (wu-sheng) [inline docs/en/changes/changes.md:107]: We don't have this yet. We only have i18n for it.

Comment (CodePrometheus) [reply docs/en/changes/changes.md]: I misunderstood before, it should be ok now.

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**